### ISSUE TITLE
getTransaction throws INVALID_ARGUMENT: hash or height is not specified

### ISSUE THREAD

**Issue by dashameter** [dashevo/wallet-lib#220]
Here is a tx on testnet: https://testnet-insight.dashevo.org/insight/tx/3e38095dbea9c628cede35577281d7e14c393781a1ae2b0cb1b6de47102e54cd

according to: https://dashevo.github.io/wallet-lib/#/account/getTransaction

```
account.getTransaction('3e38095dbea9c628cede35577281d7e14c393781a1ae2b0cb1b6de47102e54cd')
```

throws:

```
(node:373692) UnhandledPromiseRejectionWarning: Error: 3 INVALID_ARGUMENT: hash or height is not specified
```

**Comment by dashameter** [comment]
@Alex-Werner will this bug be fixed in the upcoming v.21 release?

**Comment by Alex-Werner** [comment]
In https://github.com/dashevo/wallet-lib/pull/295, we address this specific issue, basically, we try to fetch blockHeader of an unconfirmed transaction, therefore blockhash is unexistant and the blockHeader fetch fails with missing hash or height. 
We are now not performing such request within #295, but the fact that we delay the retry event to the next block is still pending. 
I think doing it in #295 was too much for a single PR, so another PR will come on top of 295 to address specifically this. 

TL;DR : As far as it is planned, yes planned for v21.

**Comment by SamKirby22** [comment]
Discussed in triage and agreed to close.
